src/parameterizations/lateral/MOM_thickness_diffuse.F90

```diff
@@ -146,17 +162,22 @@ subroutine thickness_diffuse(h, uhtr, vhtr, tv, dt, G, GV, US, MEKE, VarMix, CDp
   endif
 
   use_VarMix = .false. ; Resoln_scaled = .false. ; use_stored_slopes = .false.
-  khth_use_ebt_struct = .false.
+  khth_use_ebt_struct = .false. ; use_Visbeck = .false. ; use_QG_Leith = .false.
+
   if (associated(VarMix)) then
     use_VarMix = VarMix%use_variable_mixing .and. (CS%KHTH_Slope_Cff > 0.)
     Resoln_scaled = VarMix%Resoln_scaled_KhTh
     use_stored_slopes = VarMix%use_stored_slopes
     khth_use_ebt_struct = VarMix%khth_use_ebt_struct
+    use_Visbeck = VarMix%use_Visbeck
+    use_QG_Leith = VarMix%use_QG_Leith_GM
     if (associated(VarMix%cg1)) cg1 => VarMix%cg1
   else
     cg1 => null()
   endif
 
+  epsilon = 1.e-6
```

Comment:
This is a dimensional number so must have appropriate unit scaling. @Hallberg-NOAA  ro @marshallward can advise on how. I also think this should be a run-timeparameter since 1e-6 s-1 might not be the best number to use always.

Reply:
@Hallberg-NOAA just pointed out that 1e-6 is a large number relative to SN, not small !?